boards.txt

```diff
@@ -672,6 +672,9 @@ esp32.menu.PartitionScheme.fatflash.upload.maximum_size=2097152
 esp32.menu.PartitionScheme.app3M_fat9M_16MB=16M Flash (3MB APP/9.9MB FATFS)
 esp32.menu.PartitionScheme.app3M_fat9M_16MB.build.partitions=app3M_fat9M_16MB
 esp32.menu.PartitionScheme.app3M_fat9M_16MB.upload.maximum_size=3145728
+esp32.menu.PartitionScheme.insights=Insights
+esp32.menu.PartitionScheme.insights.build.partitions=insights
+esp32.menu.PartitionScheme.insights.upload.maximum_size=3145728
```

Comment:
this should be added to the other dev boards as well

Reply:
PTAL